src/exif.c

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 Google LLC. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "avif/internal.h"
+
+#include <stdint.h>
+#include <string.h>
+
+avifResult avifExtractExifTiffHeaderOffset(const avifRWData * exif, uint32_t * exifTiffHeaderOffset)
```

Comment:
Nit/optional: Since there is only one offset in this function, the second parameter could be named `offset` with no risk of confusion. The function name `avifExtractExifTiffHeaderOffset` documents what kind of offset it is.